apps/client/sttr_build/config.local.dist.ts

```diff
@@ -0,0 +1,17 @@
+import { apis } from "./config.ts";
```

Comment:
This file should not be added to gitignore?

Reply:
No, will add comments why this `.dist.` file is useful.